directory_sso_api_client/user.py

```diff
@@ -4,8 +4,9 @@
 class UserAPIClient(BaseAPIClient):
 
     endpoints = {
-        'session_user': 'session-user/',
-        'last_login': 'last-login/',
+        'session_user': 'api/v1/session-user/',
```

Comment:
sure, but I'll do the env var update in api once my API is using this release. If I make the change now, API will be more broken :)